plugins/nodes/src/js/columns/NodesTablePublicIPColumn.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { TextCell } from "@dcos/ui-kit";
+
+import { default as schema } from "../data/NodesResolver";
```

Comment:
The local imports should be below the non-local ones.

Reply:
I wonder what has happened to our linter? We do have this as a rule, don't we?